sdk/core/azure-core/test/ut/transport_adapter_base.hpp

```diff
@@ -19,8 +19,20 @@
 
 namespace Azure { namespace Core { namespace Test {
 
-  class TransportAdapter
-      : public testing::TestWithParam<Azure::Core::Http::TransportPolicyOptions> {
+  struct TransportAdaptersTestParameter
+  {
+    std::string Prefix;
```

Comment:
What's the benefit of prefix instead of suffix. It's fine, but I want to understand your motivation for the change.

Reply:
It should be `suffix`. I have updated it.
This is the string that gtest adds to the end of the test name and that makes each test different from the same test but with a different parameter.

This is how it looks:
![image](https://user-images.githubusercontent.com/24213737/102656066-2e869000-4128-11eb-9b94-317fc8a7113f.png)

Reply:
Does gtest allow making it a prefix for grouping reasons?